docs/src/extend/custom-formatters.md

```diff
@@ -29,37 +35,17 @@ module.exports = async function(results) {
 };
 ```
 
-To run ESLint with this formatter, you can use the `-f` (or `--format`) command line flag:
+To run ESLint with this formatter, you can use the [`-f` (or `--format`)](../use/command-line-interface#f---format) command line flag:
 
 ```bash
 eslint -f ./my-awesome-formatter.js src/
 ```
 
-In order to use a local file as a custom formatter, you must begin the filename with a dot (such as `./my-awesome-formatter.js` or `../formatters/my-awesome-formatter.js`).
-
-## Packaging the Custom Formatter
+In order to use a local file as a custom formatter, you must begin the path to the custom formatter with a period (`.`), such as `./my-awesome-formatter.js` or `../formatters/my-awesome-formatter.js`.
```

Comment:
It seems like this paragraph should come before the example on line 40 to give context?